tests/build_tests/super_errors/input.js

```diff
@@ -1,65 +1,69 @@
-const fs = require("fs");
-const path = require("path");
-const child_process = require("child_process");
+// @ts-check
 
-const { bsc_exe: bsc } = require("#cli/bin_path");
-const { normalizeNewlines } = require("../utils.js");
+import { readdirSync } from "node:fs";
+import * as fs from "node:fs/promises";
+import * as path from "node:path";
+import { setup } from "#dev/process";
+import { normalizeNewlines } from "#dev/utils";
 
-const expectedDir = path.join(__dirname, "expected");
+const { bsc } = setup(import.meta.dirname);
 
-const fixtures = fs
-  .readdirSync(path.join(__dirname, "fixtures"))
-  .filter(fileName => path.extname(fileName) === ".res");
+const expectedDir = path.join(import.meta.dirname, "expected");
 
-// const runtime = path.join(__dirname, '..', '..', 'runtime')
-const prefix = `${bsc} -w +A -bs-jsx 4`;
+const fixtures = readdirSync("fixtures").filter(
+  fileName => path.extname(fileName) === ".res",
+);
 
-const updateTests = process.argv[2] === "update";
+const prefix = ["-w", "+A", "-bs-jsx", "4"];
 
+const updateTests = process.argv[2] === "update";
 
+/**
+ * @param {string} output
+ * @return {string}
+ */
 function postProcessErrorOutput(output) {
-  output = output.trimRight();
-  output = output.replace(
+  let result = output;
+  result = result.trimEnd();
+  result = result.replace(
     /(?:[A-Z]:)?[\\/][^ ]+?tests[\\/]build_tests[\\/]super_errors[\\/]([^:]+)/g,
-    (_match, path, _offset, _string) => "/.../" + path.replace("\\", "/"),
+    (_match, path, _offset, _string) => `/.../${path.replace("\\", "/")}`,
```

Comment:
Not sure if using template strings instead of `+` makes things better in every case...
In this case I find the original version more readable.

Reply:
It is Biome's recommended rule. We can turn off or change the rule as we want.

Reply:
My personal preference would be to turn it off and, if that is not too much work, look through related changes in the PR again.

E.g., there is also

```js
`rescript_${crypto.randomBytes(8).toString("hex")}${path.parse(use_stdin).base}`
```

which I don't find particularly readable. Maybe it's just that each part should be assigned to a variable beforehand, then it would look nice.